### PR TITLE
Add tessa's changes, clean up comments/signposting, missing citations, and add abstract

### PR DESCRIPTION
**Pull request by taylorreiter**
The biggest things this PR does is:

1. address comments Tessa provided on the manuscript, including simplifying the first section to only include protein k-mers, instead of protein, hayhoff, and hp. We didn't explore dayhoff and hp much, so I took that bit out. (although I did leave the supplemental figure as is...I could cut that down to only protein as well)
2. adds an abstract

@ctb & @bluegenes ready for review. I would love if you could comment that you have seen this pull request and you are ok with the changes.

After this is merged in, and if there are no other immediate changes, I plan to send it along to the other co-authors for approval.

Once everything is approved by all co-authors, I'll generate zenodo DOIs for the companion analysis repositories & update the manuscript to reference them, update the citation to Tessa's research with a url to a draft of her paper, dump the abstract and title in the README of this repo, and transfer the repo to DIB lab. Then I think we will be ready to preprint!

I will also update the citations to charcoal, the IBD paper, and Tessa's paper once those are preprinted.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 ## Abstract {.page_break_before}
 
-
+Microbial pangenomes reflect the metabolic diversity of groups of organisms.
+As *de novo* metagenome analysis techniques have matured and reference genome databases have expanded, metapangenome analysis has risen in popularity as a tool to organize the functional potential of organisms in relation to the environment from which those organisms were sampled.
+However, the reliance on assembly and binning or on reference databases often leaves substantial portions of metagenomes unanalyzed, thereby underestimating the functional potential of a community.
+To improve this problem, we present a method for metapangenomics that relies on protein k-mers (k~aa~-mers) and metagenome assembly graph queries.
```
Comment (ctb) [inline content/01.abstract.md:6]:
```suggestion
To address this challenge, we present a method for metapangenomics that relies on protein k-mers (k~aa~-mers) and metagenome assembly graph queries.
```

don't love my suggestion, but I don't think we want to improve a problem, either 😜

File: content/04.discussion.md [hunk 2]
```diff
@@ -12,9 +12,8 @@ This may improve functional recall from metagenome short reads from organisms wi
...
 Exact matching of k-mers between genomes enables additional genomes to be added to the pangenome without having to re-cluster gene sequences. 
 Exact matching also allows direct comparisons to distantly related organisms, which creates a unified framework for genome comparisons even when organisms are distantly related.
-While we predominately explored k~aa~-mers of length 10 in this paper, k-mers from other degenerate alphabets like Dayhoff or hydrophobic-polar encodings may allow for accurate comparisons at greater evolutionary distances (CITE: Tessa).
-When pangenomes are built from FracMinHashes as performed here, the scaled down sampling parameter may also enable fast pangenome size estimation even for very large collections of genomes.
-CTB??? genes in a genome, and could be potentially used a quality control metric for annotated genomes.
+While we predominately explored k~aa~-mers of length 10 in this paper, k-mers from other degenerate alphabets like Dayhoff or hydrophobic-polar encodings may allow for accurate comparisons at greater evolutionary distances [@doi:10.1101/2021.07.09.450799].
```
Comment (ctb) [inline content/04.discussion.md:15]:
```suggestion
While we predominantly explored k~aa~-mers of length 10 in this paper, k-mers from other degenerate alphabets like Dayhoff or hydrophobic-polar encodings may allow for accurate comparisons at greater evolutionary distances [@doi:10.1101/2021.07.09.450799].
```

File: content/06.methods.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
-<!-- Can I pretty plz keep these separate or do they need to be combined? CTB: is fine :) -->
-All code is available at github.com/taylorreiter/2021-panmers (results section 1), github.com/taylorreiter/2021-orpheum-sim (results section 2), and https://github.com/taylorreiter/2021-metapangenome-example (results section 3).
+All code is available at github.com/dib-lab/2021-panmers (results section 1), github.com/dib-lab/2021-orpheum-sim (results section 2), and https://github.com/dib-lab/2021-metapangenome-example (results section 3).
 
 CTB: please provide zenodo DOIs :)
 
 ### Selection of benchmarking species for pangenome analysis
 We selected a species representative for each of the 23 phyla in GTDB rs202 [@doi:10.1093/nar/gkab776]. 
-To select representative species, we first filtered species with fewer than 20 representatives and greater than 1000 representatives.
+To select representative species, we first filtered species with fewer than 20 genomes and greater than 1000 genomes.
```
Comment (ctb) [inline content/06.methods.md:9]:
```suggestion
To select representative species, we first removed species with fewer than 20 genomes and greater than 1000 genomes.
```

File: content/06.methods.md [hunk 4]
```diff
@@ -96,7 +94,7 @@ We then constructed *de novo* metapangenomes for each species.
 Using quality controlled reads from each sample, we assembled each metagenome separately using megahit with default parameters [@doi:10.1093/bioinformatics/btv033].
 We binned the resultant assemblies using metabat2 with parameter `-m 1500` [@doi:10.7717/peerj.7359].
 We assigned GTDB species to each bin using sourmash `gather` (DNA, *k* = 31, scaled = 2000) against the GTDB rs202 database, selecting the species of the best match [@doi:10.1101/2022.01.11.475838].
-We decontaminated each bin using charcoal using default parameters (CITE: charcoal).
+We decontaminated each bin using charcoal using default parameters [@url:https://github.com/dib-lab/charcoal].
```
Comment (ctb) [inline content/06.methods.md:97]:
```suggestion
We decontaminated each bin with charcoal using default parameters [@url:https://github.com/dib-lab/charcoal].
```

**Review by ctb — APPROVED** [review]
Made some suggestions. Looks good to me!

File: content/03.results.md [hunk 5]
```diff
@@ -15,28 +15,27 @@ Approaches that were developed or tested in this manuscript are outlined in grey
...
 We compared pangenomes from 23 species belonging to 23 phyla in the GTDB taxonomy [@doi:10.1093/nar/gkab776], with pangenome size ranging from 20-972 genomes (mean = 203 genomes, median = 44 genomes) (**Figure S @fig:tree_fig**).
 For each pangenome, we compared the total number of genes to the total number of k-mers, and the number of unique genes to the number of distinct k-mers within each genome.
 We also tested the similarity of presence/absence profiles between pangenomes constructed with different methods using the Mantel test.
 
-For these three metrics, performance varied minimally across encodings and k-mer sizes, but varied dramatically between different pangenomes: both k-mers and genes are highly correlated for some pangenomes and are not correlated for others (**Figure S @fig:boxplt_fig**).
+For these three metrics, performance varied minimally across encodings and k-mer sizes, but varied dramatically for some pangenomes: both k-mers and genes are highly correlated for some pangenomes and are not correlated for others (**Figure S @fig:boxplt_fig**).
```
[the remark this answers is not in the document]
Comment (taylorreiter) [reply content/03.results.md:26]:
```suggestion
For these three metrics, performance was stable across k~aa~-mer sizes, but varied dramatically for some pangenomes: both k-mers and genes are highly correlated for some pangenomes and are not correlated for others (**Figure S @fig:boxplt_fig**).
```

**Comment by taylorreiter** [comment]
Thanks @ctb! i've committed your changes

File: content/06.methods.md [hunk 6]
```diff
@@ -54,20 +53,19 @@ We used the default short read error profile within polyester.
 ### Determining short read open reading frames with orpheum 
 We used the orpheum tool to predict open reading frames from simulated short reads [@doi:10.1101/2021.07.09.450799].
 Orpheum was developed to predict open reading frames in short RNA-seq reads from Eukaryotic organisms without a reference genome or transcriptome sequence [@doi:10.1101/2021.07.09.450799].
-Orpheum perform six-frame translation on nucleotide sequencing reads, calculates k-mers in an amino acid, dayhoff, or hydrophobic-polar encoding at the designated k-mer length, and then estimates the Jaccard similarity between k-mers in each translation frame and a database. 
+Orpheum perform six-frame translation on nucleotide sequencing reads, calculates k-mers in an amino acid encoding at the designated k-mer length, and then estimates the Jaccard similarity between k-mers in each translation frame and a database.
```
Comment (bluegenes) [inline content/06.methods.md:56]: Containment (of ref k-mers in each translated frame)

**Review by bluegenes — APPROVED** [review]

File: content/06.methods.md (hunk 6 above)
Comment (taylorreiter) [reply content/06.methods.md:56]:
```suggestion
Orpheum performs six-frame translation on nucleotide sequencing reads, calculates k-mers in an amino acid encoding at the designated k-mer length, and then estimates the containment of k-mers in a reference database in each translated frame. 
```

**Comment by taylorreiter** [comment]
Thank you both! merging now.

**Pull request merged**